Library/OptimizelyExperiment.swift

```diff
@@ -3,6 +3,7 @@ import Foundation
 public enum OptimizelyExperiment {
   public enum Key: String {
     case pledgeCTACopy = "pledge_cta_copy"
+    case categoryPersonalization = "onboarding_category_personalization_flow"
```

Comment:
With our other experiments we've camelCased the snake_cased value as the case name.

Reply:
It's a little verbose, but ok.